### PR TITLE
Add portuguese

### PR DESCRIPTION
**Pull request by tsubik**
+ new task to fill translations table with fallback entries.

### CODE REVIEW

File: config/initializers/locale.rb [hunk 1]
```diff
@@ -1,4 +1,4 @@
 # frozen_string_literal: true
 
-I18n.available_locales = [:en, :fr, 'zh-CN', :ja, :ko, :vi]
+I18n.available_locales = [:en, :fr, :pt, 'zh-CN', :ja, :ko, :vi]
```
Comment (santostiago) [inline config/initializers/locale.rb:3]: it's `pt`? Not `pt-pt`  or `pt-br`?

**Comment by tsubik** [comment]
Well, I didn't ask tbh, just did the same as for English or French. I think
that does not matter that much, right?

**Review by santostiago — APPROVED** [review]
👍

**Comment by tsubik** [comment]
At least in our case.

**Pull request merged**